tiledb/bioimg/converters/base.py

```diff
@@ -191,10 +190,16 @@ def to_tiledb(
         :param register_kwargs: Cloud group registration optional args e.g namespace,
             parent_uri, storage_uri, credentials_name
         :param reader_kwargs: Keyword arguments passed to the _ImageReaderType constructor.
-        :param generate_pyramid: If true, generate downsampled levels keeping only level 0 of the original image.
-        :param pyramid_mode: The scaling approach used. Options other than the default may require more system memory.
-        :param pyramid_scale: The downsample ratios of the generated levels.
-        :param interpolation_method: The intrepolation method for downsampling the base image.
+        :param pyramid_kwargs: Keyword arguments passed to the scaler constructor for generating downsampled versions of
+            the base level.
+            **Note**: Valid keyword arguments are:
+            scale_factors (Required): The downsample factor for each level
+            scale_axes (Required): The axes which will be downsampled
+            chunked (Optional): Default True. If true the image is split into chunks and each one is independently
+            downsampled. If false the entire image is downsampled at once, but it requires more memory.
+            progressive (Optional): Default True. If true each downsampled image is generated using the previous level.
+            If false for every downsampled image the level_min is used, but it requires more memory.
+            order (Optional): Default 1. The interpolation order for downsampling.
```

Comment:
Better reuse this more detailed description of the parameter: https://github.com/scikit-image/scikit-image/blob/v0.19.2/skimage/transform/_warps.py#L98-L101